services/user.py

```diff
@@ -0,0 +1,61 @@
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None,
+) -> User:
+
+    user = User.objects.create_user(
+        username=username,
+        password=password,

```

Comment:
Ensure that the `username` and `password` meet any required validation criteria before creating the user. This will help prevent potential errors and ensure data integrity.